### ISSUE TITLE
NVDA | Fails Accessibility Prospective

### ISSUE THREAD

**Issue by assach19** [Hacker0x01/react-datepicker#3264]
**Describe the bug**
We are using the "react-datepicker": "^4.2.1", this is working perfectly fine on Windows/Chrome/Jaws. The Calendar is able to be navigated with keyboard and has proper focus order as expected.

It fails for NVDA, a user is not able to choose every next date, focus for them is missing.

**To Reproduce**

1. Open NVDA to validate
2. Open the Calendar.
3. Navigate between dates.
4. At times, it's skipping the dates. Suppose moving from 16 to next date is taking the navigator to 22 and then 25.

**Expected behavior**
It shouldn't skip the dates while reading through NVDA.

**Desktop (please complete the following information):**

- OS: Windows
- Browser: Chrome NVDA
- Version:

**Comment by assach19** [comment]
NVDA Datepicker issues. Any luck with this yet?

**Comment by JablDouble** [comment]
@assach19 It seems that I found the solution. Just use alt + arrow when you select you date.

**Comment by assach19** [comment]
thanks @JablDouble. That's how we closed our issue and got it closed by third party vendor accepting that alt+arrow key should be fine.

Closing this thread.
